recipes/modsecurity/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "modsecurity" %}
+{% set version = "3.0.2" %}
+{% set sha256 = "e9d7f4f33ddfc3335073f8c5efa7b6d83bd532535b6b836a300c60fc843d780b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SpiderLabs/ModSecurity/releases/download/v{{ version }}/modsecurity-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - configure.ac.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf
+    - libtool
+    - automake
+    - make
+    - pkg-config
+    - git
+  host:
+    - bison
+    - pcre
+    - yajl
+    - curl
+    - geoip
+    - flex
+    - libxml2
+  run:
+    - bison
+    - pcre
+    - yajl
+    - curl
+    - geoip
+    - flex
+    - libxml2
+
+test:
+  commands:
+    - test -f "$PREFIX/include/modsecurity/modsecurity.h"
+    - test -f "$PREFIX/lib/libmodsecurity.so"
+    - test -f "$PREFIX/lib/libmodsecurity.a"
+
+about:
+  home: https://www.modsecurity.org/
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE.txt
+  summary: 'modsecurity is one component of the ModSecurity v3 project. The library codebase serves as an interface to ModSecurity Connectors taking in web traffic and applying traditional ModSecurity processing.'
+
+  description: |
+    modsecurity is one component of the ModSecurity v3 project. The library codebase serves as an interface to ModSecurity Connectors taking in web traffic and applying traditional ModSecurity processing. In general, it provides the capability to load/interpret rules written in the ModSecurity SecRules format and apply them to HTTP content provided by your application via Connectors.
+  doc_url: https://github.com/SpiderLabs/ModSecurity/tree/v3/master/doc
+  dev_url: https://github.com/SpiderLabs/ModSecurity
+
+extra:
+  recipe-maintainers:
+    - GustavoKatel
+    - Davysson0
```

Comment:
@Davysson0 are you okay with being a maintainer on this recipe?

Reply:
Sure